lib/tropic/tropic2form.flow

```diff
@@ -309,10 +309,14 @@ tropic2Acc(b : Tropic, parent : TParentInfo, sheet : Stylesheet, metricsOnly : b
 		TDynamicGroup2(stackChanges, currentStack, combiner) : {
 			fgroupChanges : DynamicBehaviour<[FGroupChange]> = make([]);
 			tropicItems : DynamicBehaviour<[TDynamicGroupItem]> = make([]);
-			verticalCombiner = if (combiner == TLinesCombiner()) FAddition() else FMax();
-			rtl = combiner == TColsACombiner() && getDefaultRtl();
+			verticalCombiner = if (combiner == TLinesCombiner() || combiner == TLinesACombiner()) FAddition() else FMax();
+			rtl = getDefaultRtl();
```

Comment:
I do not remember exactly what for, but surely this condition (rtl) was added for a reason. Did you ensure all combiners work correctly?